### PR TITLE
feat: use frontend-platform for runways

### PR DESCRIPTION
**Pull request by abutterworth**
Move logging, analytics, auth, base, and i18n to implementations in frontend-platform.

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -15,31 +18,35 @@ import configureStore from './data/configureStore';
...
 import './assets/favicon.ico';
 
-App.subscribe(APP_READY, () => {
+
+
+subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider store={configureStore()}>
-      <Header />
+
```
Comment (davidjoy) [inline src/index.jsx]: fix meeeeeeeeeee

File: src/profile/ProfilePage.test.jsx [hunk 2]
```diff
@@ -33,21 +36,51 @@ const requiredProfilePageProps = {
 // Mock language cookie
 Object.defineProperty(global.document, 'cookie', {
   writable: true,
-  value: `${App.config.LANGUAGE_PREFERENCE_COOKIE_NAME}=en`,
+  value: `${getConfig().LANGUAGE_PREFERENCE_COOKIE_NAME}=en`,
 });
-App.apiClient = jest.fn();
 
-configureI18n(App.config, messages);
+const mockAxios = new MockAdapter(axios);
```
Comment (davidjoy) [inline src/profile/ProfilePage.test.jsx]: Maybe don't need this?

File: src/profile/data/sagas.js [hunk 3]
```diff
@@ -177,7 +177,7 @@ export function* handleDeleteProfilePhoto(action) {
 }
 
 export function handleFetchUserAccountFailure(action) {
```
Comment (davidjoy) [inline src/profile/data/sagas.js]: Delete handleFetchUserAccount completely

File: src/profile/data/sagas.test.js [hunk 4]
```diff
@@ -81,11 +81,12 @@ describe('RootSaga', () => {
         username: 'gonzo',
         other: 'data',
       };
+      getAuthenticatedUser.mockReturnValue(userAccount);
       const selectorData = {
         userAccount,
       };
 
-      App.authenticatedUser.username = 'gonzo';
+      // App.authenticatedUser.username = 'gonzo';
```
Comment (davidjoy) [inline src/profile/data/sagas.test.js]: delete me

File: src/index.jsx (hunk 1 above)
Comment (abutterworth) [reply src/index.jsx]: fiiiiiinnnnne

File: src/profile/ProfilePage.test.jsx (hunk 2 above)
Comment (abutterworth) [reply src/profile/ProfilePage.test.jsx]: Didn't need it

File: src/profile/data/sagas.js (hunk 3 above)
Comment (abutterworth) [reply src/profile/data/sagas.js]: Done

File: src/profile/data/sagas.test.js (hunk 4 above)
Comment (abutterworth) [reply src/profile/data/sagas.test.js]: done

**Comment by abutterworth** [comment]
Before merge, needs to use published versions of header and footer. Right now it points at github.

**Pull request merged**